pcsx2-qt/Settings/InputBindingWidget.cpp

```diff
@@ -74,9 +74,15 @@ void InputBindingWidget::initialize(
 
 void InputBindingWidget::updateText()
 {
+	const QString binding_tip(tr("\n\nLeft click to assign a new button\nShift + left click for additional bindings"));
```

Comment:
Probably best to make this one message, in case some locales want a different order.